apps/evm/src/clients/api/queries/useGetPools/getPools/index.ts

```diff
@@ -22,7 +22,7 @@ export const getPools = async ({
   tokens,
 }: GetPoolsInput) => {
   const [
-    { pools: apiPools },
+    { pools: apiPools, tokenMetadata: apiTokenMetadata },
```

Comment:
Sorry if you talked about this before and I missed it, but why don't we keep using the `tokenPriceCents` property of the `ApiMarket` type? The frontend only uses the underlying token price coming from the oracle (as highlighted by the logic you add to the `getPools/formatOutput` function in this PR).

If the reason is that we want to use a different price source for distributions, we can still do that by reusing the `priceMantissa` field of the `ApiReward` type (here as well the backend would make the connection so that we don't need any extra logic on the frontend).